dart/packages/fury-core/lib/src/excep/ser/ser_range_excep.dart

```diff
@@ -0,0 +1,25 @@
+import 'package:fury_core/src/const/obj_type.dart';
+import 'package:fury_core/src/excep/ser/serialization_excep.dart';
```

Comment:
`ser/deser` seems not be common abbr, could we sue complete name in the codebase?